releasenotes/notes/update-pulse-gate-pass-for-target-ebfb0ec9571f058e.yaml

```diff
@@ -0,0 +1,25 @@
+---
+features:
+  - |
+    A new method :meth:`.CalibrationEntry.user_provided` has been added to
+    calibration entries. This method can be called to check whether the entry
+    is defined by an end user or backend.
+  - |
+    New method :meth:`.Target.calibration_supported` and
+    :meth:`.Target.get_calibration` have been added to provide
+    convenient access to the calibration of instruction.
+    The getter method can be called with parameter args and kwargs, and
+    it returns a pulse schedule built with parameters when the calibration
+    is templated with parameters.
+upgrade:
+  - |
+    :meth:`.Target.update_from_instruction_schedule_map` no longer raises
+    KeyError nor ValueError when qubits are missing in the target instruction
+    or inst_name_map is not provided for undefined instruction.
+    In the former case, it just ignores the inst map definition for undefined qubits.
+    In the latter case, gate mapping is pulled from the standard Qiskit gates
+    and finally custom :class:`.Gate` object is defined from the schedule name.
+fixes:
+  - |
+    A bug that a custom calibration in :class:`.InstructionScheduleMap` is ignored
+    when it is set to transpile with :class:`.BackendV2` instance.
```

Comment:
Is this just #9489 or is there a larger bug being fixed here? If it's just #9489 we don't need a release note because we've never released that issue.

Reply:
Thanks. Removed in [bdb790a](https://github.com/Qiskit/qiskit-terra/pull/9587/commits/bdb790af888455c2f31d5fa7d111a9d697798796)